security-oauth2/src/main/java/io/micronaut/security/oauth2/endpoint/authorization/response/AuthorizationErrorResponseExceptionHandler.java

```diff
@@ -31,8 +34,34 @@
 @Singleton
 public class AuthorizationErrorResponseExceptionHandler implements ExceptionHandler<AuthorizationErrorResponseException, MutableHttpResponse<?>> {
 
+    private final ErrorResponseProcessor<?> responseProcessor;
+
+    /**
+     * Default constructor.
+     */
+    public AuthorizationErrorResponseExceptionHandler() {
```

Comment:
add javadoc `@deprecated` pointing user to alternative.

Reply:
done